app-directory/LICENSE

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
This should DEFINITELY not be here. There is a license for FDC3, and it is not copyrighted by IHS Markit.